doc/modules/language-guide/pages/about-this-guide.adoc

```diff
@@ -77,7 +77,6 @@ As a counterpoint to the core values and goals, the engineering organization als
 
 . Having a more advanced type system, with more complex types.
 . Simplicity over functionality in design or implementation (the "Worse is Better" approach).
-. Interoperability with existing smart contract platforms.
```

Comment:
I would not remove it, but rephrase it as
```suggestion
 . Running on other smart contract platforms.
```
(because “interop” in the sense of sending messages to might be in scope once it is in in scope for the IC)